packages/contracts-bedrock/deploy/020-SystemDictatorSteps-1.ts

```diff
@@ -175,10 +173,9 @@ const deployFn: DeployFunction = async (hre) => {
       console.log(`Please transfer L1ERC721Bridge (proxy) owner to MSD`)
       console.log(`L1ERC721BridgeProxy address: ${L1ERC721BridgeProxy.address}`)
       console.log(`MSD address: ${SystemDictator.address}`)
-      console.log(`JSON:`)
-      console.log(jsonifyTransaction(tx))
-      console.log(getCastCommand(tx))
-      console.log(await getTenderlySimulationLink(SystemDictator.provider, tx))
+      printJsonTransaction(tx)
+      printCastCommand(tx)
+      await printTenderlySimulationLink(SystemDictator.provider, tx)
```

Comment:
another cleanup would be query the chainid once at the top of the script and then pass the chainid in instead of the provider